src/main/java/uk/gov/hmcts/reform/hmc/model/hearing/RequirementType.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.reform.hmc.model.hearing;
+
+public enum RequirementType {
+    MUSTINC,
```

Comment:
I think MUST_INCLUDE and OPTIONAL_INCLUDE, or MANDATORY, OPTIONAL will be easier to understand when used in the code, especially if not qualified

Reply:
This is existing code pulled over from civil-service repo into common place. not changing this here.